cves/CVE-2014-0483.yml

```diff
@@ -141,194 +180,181 @@ unit_tested:
     Were automated unit tests involved in this vulnerability?
     Was the original code unit tested, or not unit tested? Did the fix involve
     improving the automated tests?
-
     For code: and fix: - your answer should be boolean.
-
     For the code_answer below, look not only at the fix but the surrounding
     code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
-
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: true
```

Comment:
For your answers you need to explain how you came to the conclusion that the OG code or fix was tested.